src/WorldBank/WorldBankProvider.fs

```diff
@@ -24,7 +24,7 @@ type public WorldBankProvider(cfg:TypeProviderConfig) as this =
     let asm = System.Reflection.Assembly.GetExecutingAssembly()
     let ns = "FSharp.Data" 
 
-    let defaultServiceUrl = "http://api.worldbank.org"
+    let defaultServiceUrl = "http://api.worldbank.org/v2"
```

Comment:
Maybe change to `https` whilst you're at it?

Reply:
lets do in separate PR